### PR TITLE
Support multiple uses of --rosa

### PR DESCRIPTION
**Pull request by oharan2**
Rosa flag is not referring the addon itself.
Would like to install only specific addon via rosa cli.
The changes supporting --rosa addon_name usage and passing rosa to be False/True accordinally.

### CODE REVIEW

File: app/addon_cmds/commands.py [hunk 1]
```diff
@@ -56,7 +60,7 @@ def run_action(
     help="""
     \b
     Addons to install.
-    Format to pass is 'addon_name_1|param1=1,param2=2'\b
+    Format to pass is 'addon_name_1|param1=1,param2=2\'
```
Comment (myakove) [inline app/addon_cmds/commands.py:63]: I think the extra `\` in the end is not needed.

File: app/addon_cmds/commands.py [hunk 2]
```diff
@@ -109,9 +113,14 @@ def run_action(
...
     "--rosa",
-    help="Install/uninstall addons via ROSA cli",
-    show_default=True,
-    is_flag=True,
+    help="""
+    \b
+    Install/uninstall addons via ROSA cli.
+    Specify addon with addon name.
+    Example:
+    \'-a addon_name_1 -a addon_name_2 --rosa addon_name_2\'; Addon_name_2 will be installed with ROSA.
```
Comment (myakove) [inline app/addon_cmds/commands.py:121]: why the `\` needed?

File: app/addon_cmds/commands.py [hunk 3]
```diff
@@ -109,9 +113,14 @@ def run_action(
 )
 @click.option(
     "--rosa",
-    help="Install/uninstall addons via ROSA cli",
-    show_default=True,
-    is_flag=True,
+    help="""
```
Comment (myakove) [inline app/addon_cmds/commands.py:116]: Please also update the README with some examples

File: app/addon_cmds/commands.py [hunk 4]
```diff
@@ -159,6 +168,12 @@ def addon(
         )
 
     ctx.obj["addons_dict"] = addons_dict
+    if any(addon_name not in addons_dict.keys() for addon_name in rosa):
+        click.echo(
+            "An addon indicated with --rosa does not match any of addons names that were given.\nPlease fix "
+            "typo."
```
Comment (myakove) [inline app/addon_cmds/commands.py:174]: Please remove `Please fix typo.` from the error
Please log the passed addons as well

File: app/addon_cmds/commands.py [hunk 5]
```diff
@@ -135,7 +144,7 @@ def addon(
     ctx.obj["parallel"] = parallel
     ctx.obj["brew_token"] = brew_token
     ctx.obj["api_host"] = api_host
-    ctx.obj["rosa"] = rosa
+    ctx.obj["rosa"] = list(rosa)
```
Comment (myakove) [inline app/addon_cmds/commands.py:147]: Any reason to cast the tuple to list here?

**Review by myakove — CHANGES_REQUESTED** [review]

File: app/addon_cmds/commands.py (hunk 4 above)
Comment (oharan2) [reply app/addon_cmds/commands.py]: Done

File: app/addon_cmds/commands.py (hunk 3 above)
Comment (oharan2) [reply app/addon_cmds/commands.py:116]: Done

**Review by myakove — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**